classes/class4/homework-soln/public/stylesheets/styles.css

```diff
@@ -0,0 +1,60 @@
+/* defaults */
+body {
+	margin: 0;
+}
+
+p {
+	margin: 15px 0;
+}
+
+/* nav bar */
+div#nav {
+	color: #666;
+	padding: 10px 0;
+	background-color: #ddd;
+}
+	div#nav a {
+		color: #666;
+		margin-right: 10px;
+	}
+		div#nav a:visited {
+			color: #666;
+		}
+		div#nav a:hover {
+			color: #444;
+		}
+	div#nav h1 {
+		display: inline-block;
+		margin: 0 20px 0 15px;
+	}
+
+/* main div */
+div#main {
+	margin-left: 15px;
+}
+
+/* order page */
+form#order p {
+	margin: 20px 0;
+}
+	form#order p.ingredient {
+		margin: 5px 0;
+	}
+
+/* kitchen page, ingredient page */
+form.kitchen input,
+form.inStock input,
+form.outOfStock input,
+form#newIngredient input.submit {
+	float: right;
```

Comment:
I might bring the form a bit closer together. On a big screen the buttons are pretty far from the text.